docs/CONTRIBUTING.md

```diff
@@ -27,10 +29,28 @@ Some helpful guides:
 An easier solution would be to use Docker.
 
 **Setup**
+1. Clone the repository with `git clone https://github.com/Seneca-CDOT/telescope.git`
 1. Navigate to the root directory of telescope.
 1. Run `npm install`.
+1. Depending on your installation configuration, you may have to start the redis-server before proceeding to the next step. Do this by running the command `redis-server` in a seperate command window.
 1. Run `npm start` to start telescope.
-*If you get a series of errors, you may have to start redis-server depending on your installation configuration, do this by running the command `redis-server` in a seperate command window).*
+
+### Common Errors
+**Linux**
+
+If you encounter this error when running redis-server:
+
+`# Could not create server TCP listening socket *:6379: bind: Address already in use`
+
+You can run `sudo systemctl stop redis` to stop redis and try running `redis-server` again. If this doesn't work, you can check and manually kill the instance using port 6379 by running
+
+`ps aux | grep redis`
+
+```
+cindy     2570  0.0  0.0 253100  4908 pts/1    Sl+  13:11   0:00 redis-server *:6379
+cindy     2588  0.0  0.0 215744   888 pts/0    S+   13:16   0:00 grep --color=auto redis
+```
+Run `sudo kill -9 2570`
```

Comment:
This part is a bit confusing, I think it'd be better to test if the service is running (`sudo systemctl status redis`) before trying to stop it (`sudo systemctl stop redis`). Also, you can delete your user name from the example.